onnxruntime/core/providers/cpu/tensor/upsamplebase.h

```diff
@@ -174,6 +198,19 @@ class UpsampleBase {
               UpsampleModeNN + "(default) or " + UpsampleModeLinear + " or " + UpsampleModeCubic + ".");
   }
 
+  AspectRatioPolicy StringToKeepAspectRatioPolicy(const std::string& policy) {
+    if (policy == "stretch") {
+      return AspectRatioPolicy::STRETCH;
+    }
+    if (policy == "not_larger") {
+      return AspectRatioPolicy::NOT_LARGER;
+    }
+    if (policy == "not_smller") {
```

Comment:
A small typo - this should be `not_smaller`

Reply:
Good catch. Thanks